core/templates/pages/exploration-editor-page/modal-templates/lost-changes-modal.component.ts

```diff
@@ -54,11 +57,11 @@ export class LostChangesModalComponent
   }
 
   exportChangesAndClose(): void {
-    let lostChangesData: HTMLElement = (
+    let lostChangesData = (
       this.elRef.nativeElement.getElementsByClassName(
-        'oppia-lost-changes'));
-    let blob = new Blob([lostChangesData[0].innerText], {type: 'text/plain'});
-    var elem = this.windowRef.nativeWindow.document.createElement('a');
+        'oppia-lost-changes'))[0] as HTMLInputElement;
```

Comment:
Please add a comment for explicit casting here

Reply:
Done.